certified-connectors/smapOne/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2711 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "smapOne",
+    "description": "Connector to use and get data in smapOne.",

```

Comment:
Please provide a more informative and descriptive description that provides users a better understanding of the functionalities of this connector.

Reply:
Hey @souchoudhary,

i updated the description as you wished. 
"smap" is a fixed term our users know about.